build.sbt

```diff
@@ -251,7 +251,7 @@ lazy val algebirdBijection = module("bijection").settings(
 ).dependsOn(algebirdCore, algebirdTest % "test->test")
 
 lazy val algebirdSpark = module("spark").settings(
-    libraryDependencies += "org.apache.spark" %% "spark-core" % "1.3.0" % "provided",
+    libraryDependencies += "org.apache.spark" % "spark-core_2.10" % "1.3.0" % "provided",
```

Comment:
why is this change made?

Reply:
I had a problem with ```%%``` which instructs sbt to substitute current scala version to artifact name so I set the version of spark-core by myself, leaving the library version.
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.apache.spark#spark-core_2.12;1.3.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] Resolving org.scala-lang#scala-library;2.12.1 ...
```

Reply:
We have to leave this as is, or we can't publish this module for 2.10 and 2.11. Note this here:

https://github.com/twitter/algebird/blob/develop/build.sbt#L255

should prevent spark from being set up for 2.12.